MoviesStoreApp/MoviesStoreMVVMTests/TestMoviesListViewModel.swift

```diff
@@ -0,0 +1,91 @@
+// TestMoviesListViewModel.swift
+// Copyright © Aleksandr Shchepelin. All rights reserved.
+
+@testable import MoviesStoreMVVM
+import XCTest
+
+/// Тест экрана со списком фильмов + network service
+final class TestMoviesListViewModel: XCTestCase {
+    // MARK: - Private Properties
+
+    private var movies: [Movie] = []
+    private var movieInfo: MovieInfo?
+    private var keychainService = MockKeyChainService()
+    private var imageService = MockImageService()
+    private var coreDataService = MockCoreDataService()
+    private var networkService = MockNetworkService()
+    private var moviesListViewModel: MoviesListViewModelProtocol?
+    private var movieAPIService = MockMoviesAPIService()
+    private var movieCategory: MovieCategory = .popular
+
+    // MARK: - Public Methods
+
+    override func setUp() {
```

Comment:
```suggestion
    override func setUp() {
```
Добавь по проекту везде  super